### ISSUE TITLE
[Bug]: Crash On Start-Up (Exit Code 1)

### ISSUE THREAD

**Issue by KiryanEnderov** [GTMEGA/EndlessIDs#206]
### Game log
```
[10:50:57] [main/INFO]: Loading tweak class name cpw.mods.fml.common.launcher.FMLTweaker
[10:50:57] [main/INFO]: Using primary tweak class name cpw.mods.fml.common.launcher.FMLTweaker
[10:50:57] [main/INFO]: Calling tweak class cpw.mods.fml.common.launcher.FMLTweaker
[10:50:57] [main/INFO]: Forge Mod Loader version 7.99.40.1614 for Minecraft 1.7.10 loading
[10:50:57] [main/INFO]: Java is Java HotSpot(TM) 64-Bit Server VM, version 1.8.0_421, running on Windows 10:amd64:10.0, installed at C:\Program Files\Java\jre1.8.0_421
[10:50:57] [main/INFO]: Loading tweaker org.spongepowered.asm.launch.MixinTweaker from +unimixins-all-1.7.10-0.1.17.jar
[10:50:57] [main/INFO]: Loading tweaker org.spongepowered.asm.launch.MixinTweaker from [1.7.10] Flan's Mod NPC Vehicles-6.0.4.jar
[10:50:57] [main/WARN]: The coremod naruto1310.craftableAnimals.vanilla.CAFMLPlugin does not have a MCVersion annotation, it may cause issues with this version of Minecraft
[10:50:57] [main/WARN]: The coremod mods.defeatedcrow.asm.AppleMilkCorePlugin does not have a MCVersion annotation, it may cause issues with this version of Minecraft
[10:50:57] [main/WARN]: The coremod com.outlook.siribby.babyanimals.core.BabyAnimalsCore does not have a MCVersion annotation, it may cause issues with this version of Minecraft
[10:50:57] [main/INFO]: Loading tweaker org.spongepowered.asm.launch.MixinTweaker from campfirebackport-1.7.10-1.7.3.jar
[10:50:57] [main/INFO]: Loading tweaker org.spongepowered.asm.launch.MixinTweaker from chunkapi-mc1.7.10-0.5.1.jar
[10:50:58] [main/WARN]: The coremod cofh.asm.LoadingPlugin does not have a MCVersion annotation, it may cause issues with this version of Minecraft
[10:50:58] [main/INFO]: Loading tweaker org.spongepowered.asm.launch.MixinTweaker from CustomNPC-Plus-1.9.2.jar
[10:50:58] [main/INFO]: Loading tweaker org.spongepowered.asm.launch.MixinTweaker from DMod-1.7.10-1.0.3.5.jar
[10:50:58] [main/INFO]: Loading tweaker org.spongepowered.asm.launch.MixinTweaker from endlessids-mc1.7.10-1.5.4.jar
[10:50:58] [main/INFO]: Removing skill issues...
[10:50:58] [main/INFO]: Scanning for deps...
[10:50:59] [main/WARN]: Skipping non-directory, nor jar source: file:/C:/Users/USERNAME/AppData/Roaming/.minecraft/mods/__rzi_0.758
[10:51:00] [main/WARN]: Skipping non-directory, nor jar source: file:/C:/Users/USERNAME/AppData/Roaming/.minecraft/mods/__rzi_0.405
[10:51:00] [main/WARN]: Skipping non-directory, nor jar source: file:/C:/Users/USERNAME/AppData/Roaming/.minecraft/mods/__rzi_0.191
[10:51:00] [main/WARN]: Skipping non-directory, nor jar source: file:/C:/Users/USERNAME/AppData/Roaming/.minecraft/mods/Trollebas_Pack-1.2.2_on_35_RTM.zip
[10:51:02] [main/WARN]: Skipping non-directory, nor jar source: file:/C:/Users/USERNAME/AppData/Roaming/.minecraft/mods/__rzi_0.774
[10:51:02] [main/WARN]: Skipping non-directory, nor jar source: file:/C:/Users/USERNAME/AppData/Roaming/.minecraft/mods/__rzi_0.410
[10:51:02] [main/WARN]: Skipping non-directory, nor jar source: file:/C:/Users/USERNAME/AppData/Roaming/.minecraft/mods/__rzi_0.530
[10:51:02] [main/WARN]: Skipping non-directory, nor jar source: file:/C:/Users/USERNAME/AppData/Roaming/.minecraft/mods/__rzi_0.658
[10:51:02] [main/WARN]: Skipping non-directory, nor jar source: file:/C:/Users/USERNAME/AppData/Roaming/.minecraft/mods/__rzi_0.789
[10:51:02] [main/WARN]: Skipping non-directory, nor jar source: file:/C:/Users/USERNAME/AppData/Roaming/.minecraft/mods/tropicraftv6_0_5+for+MC+v1_7_10.zip
[10:51:03] [main/WARN]: Skipping non-directory, nor jar source: file:/C:/Users/USERNAME/AppData/Roaming/.minecraft/mods/DrZharks%20MoCreatures%20Mod%20v6.3.1.zip
[10:51:03] [main/WARN]: Skipping non-directory, nor jar source: file:/C:/Users/USERNAME/AppData/Roaming/.minecraft/mods/Fox%20Mod.zip
[10:51:03] [main/WARN]: Skipping non-directory, nor jar source: file:/C:/Users/USERNAME/AppData/Roaming/.minecraft/mods/Russian_Road_Pack_v1.10.zip
[10:51:03] [main/WARN]: Skipping non-directory, nor jar source: file:/C:/Users/USERNAME/AppData/Roaming/.minecraft/mods/__rzi_0.465
[10:51:04] [main/WARN]: Skipping non-directory, nor jar source: file:/C:/Users/USERNAME/AppData/Roaming/.minecraft/mods/ruby_mod_1.7.10.zip
[10:51:04] [main/WARN]: Skipping non-directory, nor jar source: file:/C:/Users/USERNAME/AppData/Roaming/.minecraft/mods/Warg%20Armor.zip
[10:51:04] [main/WARN]: Skipping non-directory, nor jar source: file:/C:/Users/USERNAME/AppData/Roaming/.minecraft/mods/__rzi_0.390
[10:51:05] [main/WARN]: Skipping non-directory, nor jar source: file:/C:/Users/USERNAME/AppData/Roaming/.minecraft/mods/__rzi_0.969
[10:51:05] [main/WARN]: Skipping non-directory, nor jar source: file:/C:/Users/USERNAME/AppData/Roaming/.minecraft/mods/__rzi_0.729
[10:51:05] [main/WARN]: Skipping non-directory, nor jar source: file:/C:/Users/USERNAME/AppData/Roaming/.minecraft/mods/__rzi_0.168
[10:51:05] [main/WARN]: Skipping non-directory, nor jar source: file:/C:/Users/USERNAME/AppData/Roaming/.minecraft/mods/RRP_roadsigns.zip
[10:51:06] [main/WARN]: Skipping non-directory, nor jar source: file:/C:/Users/USERNAME/AppData/Roaming/.minecraft/mods/Random%20Stuff%20Mod%204.0.zip
[10:51:06] [main/WARN]: Skipping non-directory, nor jar source: file:/C:/Users/USERNAME/AppData/Roaming/.minecraft/mods/__rzi_0.708
[10:51:07] [main/INFO]: [java.lang.Throwable$WrappedPrintStream:println:-1]: java.lang.ExceptionInInitializerError
[10:51:07] [main/INFO]: [java.lang.Throwable$WrappedPrintStream:println:-1]: 	at java.lang.Class.forName0(Native Method)
[10:51:07] [main/INFO]: [java.lang.Throwable$WrappedPrintStream:println:-1]: 	at java.lang.Class.forName(Unknown Source)
[10:51:07] [main/INFO]: [java.lang.Throwable$WrappedPrintStream:println:-1]: 	at cpw.mods.fml.relauncher.CoreModManager.loadCoreMod(CoreModManager.java:460)
[10:51:07] [main/INFO]: [java.lang.Throwable$WrappedPrintStream:println:-1]: 	at cpw.mods.fml.relauncher.CoreModManager.discoverCoreMods(CoreModManager.java:389)
[10:51:07] [main/INFO]: [java.lang.Throwable$WrappedPrintStream:println:-1]: 	at cpw.mods.fml.relauncher.CoreModManager.handleLaunch(CoreModManager.java:221)
[10:51:07] [main/INFO]: [java.lang.Throwable$WrappedPrintStream:println:-1]: 	at cpw.mods.fml.relauncher.FMLLaunchHandler.setupHome(FMLLaunchHandler.java:90)
[10:51:07] [main/INFO]: [java.lang.Throwable$WrappedPrintStream:println:-1]: 	at cpw.mods.fml.relauncher.FMLLaunchHandler.setupClient(FMLLaunchHandler.java:67)
[10:51:07] [main/INFO]: [java.lang.Throwable$WrappedPrintStream:println:-1]: 	at cpw.mods.fml.relauncher.FMLLaunchHandler.configureForClientLaunch(FMLLaunchHandler.java:34)
[10:51:07] [main/INFO]: [java.lang.Throwable$WrappedPrintStream:println:-1]: 	at cpw.mods.fml.common.launcher.FMLTweaker.injectIntoClassLoader(FMLTweaker.java:126)
[10:51:07] [main/INFO]: [java.lang.Throwable$WrappedPrintStream:println:-1]: 	at net.minecraft.launchwrapper.Launch.launch(Launch.java:115)
[10:51:07] [main/INFO]: [java.lang.Throwable$WrappedPrintStream:println:-1]: 	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)
[10:51:07] [main/INFO]: [java.lang.Throwable$WrappedPrintStream:println:-1]: Caused by: java.lang.SecurityException: SHA-256 digest error for assets/pixelmon/sounds/music/Across the Desert.ogg
[10:51:07] [main/INFO]: [java.lang.Throwable$WrappedPrintStream:println:-1]: 	at sun.security.util.ManifestEntryVerifier.verify(Unknown Source)
[10:51:07] [main/INFO]: [java.lang.Throwable$WrappedPrintStream:println:-1]: 	at java.util.jar.JarVerifier.processEntry(Unknown Source)
[10:51:07] [main/INFO]: [java.lang.Throwable$WrappedPrintStream:println:-1]: 	at java.util.jar.JarVerifier.update(Unknown Source)
[10:51:07] [main/INFO]: [java.lang.Throwable$WrappedPrintStream:println:-1]: 	at java.util.jar.JarInputStream.read(Unknown Source)
[10:51:07] [main/INFO]: [java.lang.Throwable$WrappedPrintStream:println:-1]: 	at java.util.zip.ZipInputStream.closeEntry(Unknown Source)
[10:51:07] [main/INFO]: [java.lang.Throwable$WrappedPrintStream:println:-1]: 	at java.util.zip.ZipInputStream.getNextEntry(Unknown Source)
[10:51:07] [main/INFO]: [java.lang.Throwable$WrappedPrintStream:println:-1]: 	at java.util.jar.JarInputStream.getNextEntry(Unknown Source)
[10:51:07] [main/INFO]: [java.lang.Throwable$WrappedPrintStream:println:-1]: 	at com.falsepattern.lib.internal.impl.dependencies.DependencyLoaderImpl.scanForDepSpecs(DependencyLoaderImpl.java:276)
[10:51:07] [main/INFO]: [java.lang.Throwable$WrappedPrintStream:println:-1]: 	at com.falsepattern.lib.internal.impl.dependencies.DependencyLoaderImpl.scanDeps(DependencyLoaderImpl.java:424)
[10:51:07] [main/INFO]: [java.lang.Throwable$WrappedPrintStream:println:-1]: 	at com.falsepattern.lib.internal.impl.dependencies.DependencyLoaderImpl.executeDependencyLoading(DependencyLoaderImpl.java:396)
[10:51:07] [main/INFO]: [java.lang.Throwable$WrappedPrintStream:println:-1]: 	at com.falsepattern.lib.internal.asm.CoreLoadingPlugin.<clinit>(CoreLoadingPlugin.java:75)
[10:51:07] [main/INFO]: [java.lang.Throwable$WrappedPrintStream:println:-1]: 	... 11 more
[10:51:07] [main/ERROR]: An error occurred trying to configure the minecraft home at C:\Users\USERNAME\AppData\Roaming\.minecraft for Forge Mod Loader
java.lang.ExceptionInInitializerError
	at java.lang.Class.forName0(Native Method) ~[?:1.8.0_421]
	at java.lang.Class.forName(Unknown Source) ~[?:1.8.0_421]
	at cpw.mods.fml.relauncher.CoreModManager.loadCoreMod(CoreModManager.java:460) ~[forge-1.7.10-10.13.4.1614-1.7.10.jar:?]
	at cpw.mods.fml.relauncher.CoreModManager.discoverCoreMods(CoreModManager.java:389) ~[forge-1.7.10-10.13.4.1614-1.7.10.jar:?]
	at cpw.mods.fml.relauncher.CoreModManager.handleLaunch(CoreModManager.java:221) ~[forge-1.7.10-10.13.4.1614-1.7.10.jar:?]
	at cpw.mods.fml.relauncher.FMLLaunchHandler.setupHome(FMLLaunchHandler.java:90) [forge-1.7.10-10.13.4.1614-1.7.10.jar:?]
	at cpw.mods.fml.relauncher.FMLLaunchHandler.setupClient(FMLLaunchHandler.java:67) [forge-1.7.10-10.13.4.1614-1.7.10.jar:?]
	at cpw.mods.fml.relauncher.FMLLaunchHandler.configureForClientLaunch(FMLLaunchHandler.java:34) [forge-1.7.10-10.13.4.1614-1.7.10.jar:?]
	at cpw.mods.fml.common.launcher.FMLTweaker.injectIntoClassLoader(FMLTweaker.java:126) [forge-1.7.10-10.13.4.1614-1.7.10.jar:?]
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:115) [launchwrapper-1.12.jar:?]
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28) [launchwrapper-1.12.jar:?]
Caused by: java.lang.SecurityException: SHA-256 digest error for assets/pixelmon/sounds/music/Across the Desert.ogg
	at sun.security.util.ManifestEntryVerifier.verify(Unknown Source) ~[?:1.8.0_421]
	at java.util.jar.JarVerifier.processEntry(Unknown Source) ~[?:1.8.0_421]
	at java.util.jar.JarVerifier.update(Unknown Source) ~[?:1.8.0_421]
	at java.util.jar.JarInputStream.read(Unknown Source) ~[?:1.8.0_421]
	at java.util.zip.ZipInputStream.closeEntry(Unknown Source) ~[?:1.8.0_421]
	at java.util.zip.ZipInputStream.getNextEntry(Unknown Source) ~[?:1.8.0_421]
	at java.util.jar.JarInputStream.getNextEntry(Unknown Source) ~[?:1.8.0_421]
	at com.falsepattern.lib.internal.impl.dependencies.DependencyLoaderImpl.scanForDepSpecs(DependencyLoaderImpl.java:276) ~[DependencyLoaderImpl.class:?]
	at com.falsepattern.lib.internal.impl.dependencies.DependencyLoaderImpl.scanDeps(DependencyLoaderImpl.java:424) ~[DependencyLoaderImpl.class:?]
	at com.falsepattern.lib.internal.impl.dependencies.DependencyLoaderImpl.executeDependencyLoading(DependencyLoaderImpl.java:396) ~[DependencyLoaderImpl.class:?]
	at com.falsepattern.lib.internal.asm.CoreLoadingPlugin.<clinit>(CoreLoadingPlugin.java:75) ~[falsepatternlib-mc1.7.10-1.4.1.jar:?]
	... 11 more
[10:51:07] [main/INFO]: [java.lang.Throwable$WrappedPrintStream:println:-1]: java.lang.ExceptionInInitializerError
[10:51:07] [main/INFO]: [java.lang.Throwable$WrappedPrintStream:println:-1]: 	at java.lang.Class.forName0(Native Method)
[10:51:07] [main/INFO]: [java.lang.Throwable$WrappedPrintStream:println:-1]: 	at java.lang.Class.forName(Unknown Source)
[10:51:07] [main/INFO]: [java.lang.Throwable$WrappedPrintStream:println:-1]: 	at cpw.mods.fml.relauncher.CoreModManager.loadCoreMod(CoreModManager.java:460)
[10:51:07] [main/INFO]: [java.lang.Throwable$WrappedPrintStream:println:-1]: 	at cpw.mods.fml.relauncher.CoreModManager.discoverCoreMods(CoreModManager.java:389)
[10:51:07] [main/INFO]: [java.lang.Throwable$WrappedPrintStream:println:-1]: 	at cpw.mods.fml.relauncher.CoreModManager.handleLaunch(CoreModManager.java:221)
[10:51:07] [main/INFO]: [java.lang.Throwable$WrappedPrintStream:println:-1]: 	at cpw.mods.fml.relauncher.FMLLaunchHandler.setupHome(FMLLaunchHandler.java:90)
[10:51:07] [main/INFO]: [java.lang.Throwable$WrappedPrintStream:println:-1]: 	at cpw.mods.fml.relauncher.FMLLaunchHandler.setupClient(FMLLaunchHandler.java:67)
[10:51:07] [main/INFO]: [java.lang.Throwable$WrappedPrintStream:println:-1]: 	at cpw.mods.fml.relauncher.FMLLaunchHandler.configureForClientLaunch(FMLLaunchHandler.java:34)
[10:51:07] [main/INFO]: [java.lang.Throwable$WrappedPrintStream:println:-1]: 	at cpw.mods.fml.common.launcher.FMLTweaker.injectIntoClassLoader(FMLTweaker.java:126)
[10:51:07] [main/INFO]: [java.lang.Throwable$WrappedPrintStream:println:-1]: 	at net.minecraft.launchwrapper.Launch.launch(Launch.java:115)
[10:51:07] [main/INFO]: [java.lang.Throwable$WrappedPrintStream:println:-1]: 	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)
[10:51:07] [main/INFO]: [java.lang.Throwable$WrappedPrintStream:println:-1]: Caused by: java.lang.SecurityException: SHA-256 digest error for assets/pixelmon/sounds/music/Across the Desert.ogg
[10:51:07] [main/INFO]: [java.lang.Throwable$WrappedPrintStream:println:-1]: 	at sun.security.util.ManifestEntryVerifier.verify(Unknown Source)
[10:51:07] [main/INFO]: [java.lang.Throwable$WrappedPrintStream:println:-1]: 	at java.util.jar.JarVerifier.processEntry(Unknown Source)
[10:51:07] [main/INFO]: [java.lang.Throwable$WrappedPrintStream:println:-1]: 	at java.util.jar.JarVerifier.update(Unknown Source)
[10:51:07] [main/INFO]: [java.lang.Throwable$WrappedPrintStream:println:-1]: 	at java.util.jar.JarInputStream.read(Unknown Source)
[10:51:07] [main/INFO]: [java.lang.Throwable$WrappedPrintStream:println:-1]: 	at java.util.zip.ZipInputStream.closeEntry(Unknown Source)
[10:51:07] [main/INFO]: [java.lang.Throwable$WrappedPrintStream:println:-1]: 	at java.util.zip.ZipInputStream.getNextEntry(Unknown Source)
[10:51:07] [main/INFO]: [java.lang.Throwable$WrappedPrintStream:println:-1]: 	at java.util.jar.JarInputStream.getNextEntry(Unknown Source)
[10:51:07] [main/INFO]: [java.lang.Throwable$WrappedPrintStream:println:-1]: 	at com.falsepattern.lib.internal.impl.dependencies.DependencyLoaderImpl.scanForDepSpecs(DependencyLoaderImpl.java:276)
[10:51:07] [main/INFO]: [java.lang.Throwable$WrappedPrintStream:println:-1]: 	at com.falsepattern.lib.internal.impl.dependencies.DependencyLoaderImpl.scanDeps(DependencyLoaderImpl.java:424)
[10:51:07] [main/INFO]: [java.lang.Throwable$WrappedPrintStream:println:-1]: 	at com.falsepattern.lib.internal.impl.dependencies.DependencyLoaderImpl.executeDependencyLoading(DependencyLoaderImpl.java:396)
[10:51:07] [main/INFO]: [java.lang.Throwable$WrappedPrintStream:println:-1]: 	at com.falsepattern.lib.internal.asm.CoreLoadingPlugin.<clinit>(CoreLoadingPlugin.java:75)
[10:51:07] [main/INFO]: [java.lang.Throwable$WrappedPrintStream:println:-1]: 	... 11 more
```
### Description

I have a lot of mods (all individual; I don't use modpacks) but I know for a fact that this crash is caused by this mod and/or its dependencies, which sucks because I need this mod to help prevent biome and potion ID conflicts. :(

**Comment by Nuclear4Hurricane** [comment]
i have same problem and it creates no crash report when i try to run it

**Comment by KiryanEnderov** [comment]
It still crashes even after updating to the latest version of FalsePatternLib.

**Comment by FalsePattern** [comment]
Fixed in FalsePatternLib 1.4.3

**Comment by KiryanEnderov** [comment]
Thanks!
